app/drivers/zephyr/battery_voltage_divider.c

```diff
@@ -205,8 +205,8 @@ static const struct bvd_config bvd_cfg = {
     .power_gpios =
         {
             DT_INST_GPIO_LABEL(0, power_gpios),
-            DT_INST_PIN(0, power_gpios),
-            DT_INST_FLAGS(0, power_gpios),
+            DT_INST_GPIO_PIN(0, power_gpios),
+            DT_INST_GPIO_FLAGS(0, power_gpios),
```

Comment:
I'll admit, I never tested this part of the code since I didn't have the hardware. This change looks proper, but it would be better suited for a separate PR from this. Could you please do this?

Reply:
Great! Sure, I've done a separate PR for it.